ticdc/manage-ticdc.md

```diff
@@ -240,21 +242,16 @@ Best practices:
 
 #### Integrate TiCDC with Kafka Connect (Confluent Platform)
 
-> **Warning:**
->
-> This is still an experimental feature. Do **NOT** use it in a production environment.
+To use the [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) provided by Confluent to stream data to relational or non-relational databases, you should use the `avro` protocol and provide a URL for [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) in `schema-registry`.
 
 Sample configuration:
 
 {{< copyable "shell-regular" >}}
 
 ```shell
---sink-uri="kafka://127.0.0.1:9092/topic-name?protocol=canal-json&kafka-version=2.4.0&protocol=avro&partition-num=6&max-message-bytes=67108864&replication-factor=1"
---opts registry="http://127.0.0.1:8081"
+--sink-uri="kafka://127.0.0.1:9092/topic-name?protocol=avro&replication-factor=3" --schema-registry="http://127.0.0.1:8081"
 ```
 
-To use the [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) provided by Confluent to stream data to relational or non-relational databases, you should use the `avro` protocol and provide a URL for [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) in `opts`. Note that the `avro` protocol and Confluent integration are **experimental**.
-
 For detailed integration guide, see [Quick Start Guide on Integrating TiDB with Confluent Platform](/ticdc/integrate-confluent-using-ticdc.md).
```

Comment:
This link should also be updated after the referred doc is updated.

Reply:
> This link should also be updated after the referred doc is updated.

I'm afraid this link will remain the same in v6.1 release, because the new doc will not be ready upon 6.1 release (Chinese draft ready on June 2nd, according to Yilin).

Reply:
Never mind. Just don't forget it and update later then.